packages/git_prompt.rb

```diff
@@ -32,7 +32,7 @@ def self.install
 
   def self.postinstall
     puts "\nTo finish the installation, execute the following:".lightblue
-    puts 'source ~/.bashrc'.lightblue
+    puts "source #{CREW_PREFIX}/etc/env.d/profile".lightblue
```

Comment:
This won't work and should be removed.  Prompt to `source ~/.bashrc` instead?  IOW, do not change.